# ROS message for 3D position of single object in camera frame

std_msgs/Header header
geometry_msgs/Pose poses
uint8 ids